api/src/pcapi/scheduled_tasks/clock.py

```diff
@@ -164,18 +166,30 @@ def pc_check_stock_quantity_consistency() -> None:
 @cron_context
 @log_cron_with_transaction
 def pc_send_yesterday_event_offers_notifications() -> None:
+    if not settings.IS_PROD and not settings.IS_TESTING:
+        return
     for educational_booking in find_educational_bookings_done_yesterday():
         send_eac_satisfaction_study_email_to_pro(educational_booking)
 
 
 @cron_context
 @log_cron_with_transaction
 def pc_send_tomorrow_events_notifications() -> None:
-    stock_ids = find_tomorrow_event_stock_ids()
+    stock_ids = find_event_stock_ids_happening_in_x_days(1)
     for stock_id in stock_ids:
         send_tomorrow_stock_notification.delay(stock_id)
 
 
+@cron_context
+@log_cron_with_transaction
+def pc_send_email_reminder_7days_before_event() -> None:
+    if not settings.IS_PROD and not settings.IS_TESTING:
+        return
+    stocks = find_event_stocks_happening_in_x_days(7)
```

Comment:
Il faudrait joinedload les offres et lieux liés ici pour charger toutes les informations liées, sinon on va faire 2 requêtes SQL par stock dans la fonction `send_reminder_7_days_before_event_to_pro`. Quelque chose comme ça (avec `import sqlalchemy.orm as sqla_orm`) : 

```suggestion
    stocks = find_event_stocks_happening_in_x_days(7).options(sqla_orm.joinedload(Stock.offer).joinedload(Offer.venue))
```

(et idem dans `commands.py` évidemment)